src/components/ui/toast.tsx

```diff
@@ -7,19 +7,33 @@ import { cn } from '../../utils'
 
 const ToastProvider = ToastPrimitives.Provider
 
+const toastViewportPositions: Record<ToastPosition, string> = {
+  "top-right": "top-0 right-0 sm:bottom-auto sm:right-0 sm:top-0",
+  "top-left": "top-0 left-0 sm:bottom-auto sm:left-0 sm:top-0",
+  "bottom-right": "bottom-0 right-0 sm:top-auto sm:right-0 sm:bottom-0",
+  "bottom-left": "bottom-0 left-0 sm:top-auto sm:left-0 sm:bottom-0",
+  center: "bottom-0 left-1/2 transform -translate-x-1/2 sm:bottom-0 sm:left-1/2 sm:transform sm:-translate-x-1/2",
+};
+
```

Comment:
What is the reason why do we have `Record<ToastPosition, string>`? 

The keys appear to be strings, except the last `center`. Are they supposed to be all strings and the record `Record<string, string>`?

Also, there is no `ToastPosition` imported nor defined anywhere

Reply:
yes I forgot to change everything back to string after I got rid of ToastPosition